File: BeautySaloon/BeautySaloon.API/Entities/BeautySaloonContextEntities/Master.cs
namespace BeautySaloon.API.Entities.BeautySaloonContextEntities;

public class Master
{
    public int? Id { get; set; }
    public string Name { get; set; }
    public List<Service>? Services { get; set; }
    public int? ScheduleId { get; set; }
    public Schedule? Schedule { get; set; }
    public List<Reservation>? Reservations { get; set; }
}

File: BeautySaloon/BeautySaloon.Angular/src/app/shared/models/master.ts
import { Reservation } from "./resrvation/reservation";
import { Schedule } from "./schedule";
import { Service } from "./service";

export interface Master {
    id: number | null;
    name: string;
    services: Service[] | null;
    scheduleId: number | null;
    schedule: Schedule | null;
    reservations: Reservation[] | null;
  }